homeassistant/components/litejet/light.py

```diff
@@ -1,43 +1,50 @@
 """Support for LiteJet lights."""
 import logging
 
-from homeassistant.components import litejet
 from homeassistant.components.light import (
     ATTR_BRIGHTNESS,
     SUPPORT_BRIGHTNESS,
     LightEntity,
 )
 
+from .const import DOMAIN
+
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_NUMBER = "number"
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
-    """Set up lights for the LiteJet platform."""
-    litejet_ = hass.data["litejet_system"]
+async def async_setup_entry(hass, config_entry, async_add_devices):
+    """Set up entry."""
+
+    system = hass.data[DOMAIN]
 
     devices = []
-    for i in litejet_.loads():
-        name = litejet_.get_load_name(i)
-        if not litejet.is_ignored(hass, name):
-            devices.append(LiteJetLight(hass, litejet_, i, name))
-    add_entities(devices, True)
+    for i in system.loads():
+        name = system.get_load_name(i)
```

Comment:
If this does I/O it needs to run in the executor.

Reply:
Looks like it does 

https://github.com/joncar/pylitejet/blob/master/pylitejet/__init__.py#L252